Fixes installation instructions & broken links, and improves readability of README (#33)
Homebrew/homebrew-linux-fonts: README.md

# Fonts for Homebrew on Linux

homebrew-linux-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts). 

> It is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux since Homebrew Casks are only for macOS.

Reviewer: I would reorder the sentences to make the last bit follow logically from what precedes it:

> homebrew-linux-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
> It is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts), since Homebrew Casks are only for macOS.

Author: ```suggestion
homebrew-linux-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
It is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts), since Homebrew Casks are only for macOS.
```

## Prerequisites

These fonts are installable with [Homebrew](http://brew.sh).

> This repository is **NOT** an "external command” of Homebrew.

Reviewer: ```suggestion
This repository is **NOT** an "external command" of Homebrew.
```

It is different from the upstream repository, `homebrew/homebrew-cask-fonts`.

> Please be aware of that.

Reviewer: What does this mean, though? In other words: as a user, this sentence makes me aware of this fact; now what am I supposed to do with this information? Are there things I would have expected to be able to do if I wasn't aware of this? If so, can we offer an example or two to make this concrete?

Author: I _think_ the intent is to additionally call out that they're separate and unequal repos: they diverge in content availability and installation mechanism. Maybe this message made more sense when cask was a separate command, which it no longer is.

I'll put up a suggestion with that in mind.

Author: https://github.com/Homebrew/homebrew-linux-fonts/pull/33/files#r1120940757

## Let’s try it!

```bash
$ brew tap homebrew/linux-fonts          # you only have to do this once!
$ brew install font-inconsolata --HEAD
```

## Working with fontconfig

If you're using major distributions in a single-user installation, run:

```
$ ln -s /home/linuxbrew/.linuxbrew/share/fonts ~/.local/share/fonts
$ fc-cache -fv
```

> For multi-user installation, you must use `sudo` to run:

Reviewer: This should say either "For multi-user installation**s**" or "For **a** multi-user installation".

Reviewer: Also, I believe this ending would read better, as it relates to the instructions above:

```suggestion
For multi-user installation, you must use `sudo` to run the commands:
```

Reviewer: Hey @colindean, please note that applying the suggestion marked this thread as resolved, but the first comment still applies.

# Fonts for Homebrew on Linux

homebrew-linux-fonts is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts). 
homebrew-linux-fonts is a Homebrew Tap which allows you to use the same friendly Homebrew-style CLI workflow for the administration of binary font files on your Linux.
It is a fork of [homebrew-cask-fonts](https://github.com/Caskroom/homebrew-cask-fonts), since Homebrew Casks are only for macOS.

## Prerequisites

> These fonts are installable with [Homebrew](http://brew.sh).
> This repository is **NOT** an "external command" of Homebrew.
> It is different from the upstream repository, `homebrew/homebrew-cask-fonts`.
> Please be aware of that.

Author: ```suggestion
These fonts are installable with [Homebrew](http://brew.sh).
Once installed, you can configure this repo as a source for fonts.

Note that this repo differs from the upstream repository, `homebrew/homebrew-cask-fonts`.
The set of fonts available may not match exactly.
Moreover, it's best to use only one of these repos on macOS;
technically this repo _and_ the upstream are usable on macOS,
but you should not use _this repo_ on macOS because upstream will have the majority of what's needed.
Changes to upstream eventually propagate to this repo.
```

Thoughts? Or too long?

Reviewer: I think lines 14–16 are not necessary, since this project already has "linux" in the name, and there's `homebrew/homebrew-cask-fonts` for macOS users anyway. Those two facts seem to me to be sufficient to signal that this is not meant for use in macOS.

Author: ```suggestion
These fonts are installable with [Homebrew](http://brew.sh).
Once installed, you can configure this repo as a source for fonts.

Note that this repo differs from the upstream repository, `homebrew/homebrew-cask-fonts`.
The set of fonts available may not match exactly.
Changes to upstream eventually propagate to this repo.
```

With 14-16 removed.

## Let’s try it!

```bash
$ brew tap homebrew/linux-fonts          # you only have to do this once!
$ brew install font-inconsolata --HEAD
```

## Working with fontconfig

If you're using major distributions in a single-user installation, run:

```
$ ln -s /home/linuxbrew/.linuxbrew/share/fonts ~/.local/share/fonts
$ fc-cache -fv
```

> For multi-user installation, you must use `sudo` to run the commands:

Author: ```suggestion
For a multi-user installation, you must use `sudo` to run the commands:
```

### Submitting a Font Cask and Bugs

This repository automatically imports new font casks from
[the upstream repository](https://github.com/homebrew/homebrew-cask-fonts/) with a bot program, so changes there
will propagate to this repository automatically eventually.

### Skipping translations by `.caskignore`

Some casks are incompatible with formula definition method.
`cask2formula` will skip translation for files listed in `.caskignore`.
`.caskignore` is written in Ruby's [`fnmatch` syntax](https://docs.ruby-lang.org/en/2.6.0/File.html#method-c-fnmatch).

## Font Licenses

> This repository is aligned with `homebrew-cask-fonts`:

Reviewer: IMO this wording is a little clearer:

```suggestion
This repository follows `homebrew-cask-fonts`'s policy regarding font licenses:
```
